.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateParser.cs

```diff
@@ -144,18 +144,40 @@ private DateTimeResolutionResult ParseBasicRegexMatch(string text, DateObject re
             foreach (var regex in this.config.DateRegexes)
             {
                 var offset = 0;
+                var swift = -2;
```

Comment:
Why -2? What does it mean? Some invalid value?
If so, why is it needed? It should also be declared in Constants so it's easier to see what it means and make sure the right value is reused where necessary.